manual/mapper/daos/defaultnullsavingstrategy/README.md

```diff
@@ -0,0 +1,33 @@
+## DefaultNullSavingStrategy
+
+If you don't specify `nullSavingStrategy` parameter on the method level you can suppress 
```

Comment:
instead of `you can suppress defaults`, I would recommend `you can adjust the default behavior`